app/viewModels/ViewArrivalMovements.scala

```diff
@@ -34,13 +34,12 @@ object ViewArrivalMovements {
     Ordering.by(identity[ChronoLocalDate])
 
   def apply(
-    movements: Seq[ViewMovement]
+    movements: Seq[ViewMovement],
+    dummmy: String = ""
```

Comment:
You can pass in a DummyImplicit here rather and remove the second argument_(implicit d: DummyImplicit)_

So long as ViewArrivalMovements case class has a single argument of Seq and apply method has the same, you're always going to get that type erasure error (the compiler doesn't know whether to use the apply within the case class or object)